Formula/step.rb

```diff
@@ -34,13 +28,19 @@ def install
       contents = Dir["{*,.git,.gitignore}"]
       (buildpath/"src/github.com/smallstep/certificates").install contents
       cd "#{buildpath}/src/github.com/smallstep/certificates" do
+        ENV["VERSION"] = "v0.8.4"
         system "make", "build"
         bin.install "bin/step-ca" => "step-ca"
       end
     end
   end
 
   test do
+    # Verify step version.
```

Comment:
We can't have tests here that needs to be updated with every release so I don't think these should be added and the existing test is good enough.

Reply:
Removed the version tests. Reasoning makes sense.